src/tests/test_scraper_base.py

```diff
@@ -1,11 +1,33 @@
-from datetime import datetime, timezone
+import os
+from datetime import datetime
 from unittest import TestCase
 from unittest.mock import MagicMock, Mock, patch
 
-from models import Platform, InitiativeImport, BatchImportState
+from models import Platform, InitiativeImport, BatchImportState, Db
 from models.initiatives import InitiativeImportState
 from platformen import Scraper
 from platformen.scraper import ScrapeException
+from tools import Geocoder
+
+
+class TestDatabase(TestCase):
+    def test_default_sqlite(self):
+        test_db = Db()
+        assert test_db.get_db_url().startswith("sqlite")
+
+    def test_postgres_from_env(self):
+        testie = {"DB_USER": "klaas", "DB_PASSWORD": "vaak", "DB_HOST": "thuis", "DB_NAME": "1"}
+        with patch.dict(os.environ, testie):
+            test_db = Db()
+            assert test_db.get_db_url().startswith("postgres")
+
+
+class TestGeocoder(TestCase):
```

Comment:
ok for now although we do really need to move the geocoder further on into the process to prevent overheating.